web/modules/custom/bnf/bnf_example_content/bnf_example_content.module

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\drupal_typed\DrupalTyped;
+
+/**
+ * Implements hook_modules_installed().
+ *
+ * @param string[] $modules
+ *   The names of the modules that were installed.
+ */
+function bnf_example_content_modules_installed(array $modules): void {
+  // Only react to our own installation.
+  if (!in_array('bnf_example_content', $modules)) {
+    return;
+  }
```

Comment:
I wonder why this is implemented in `hook_modules_installed()` instead of `hook_install()` 🤔

Reply:
Well, you tell me why you did it in `dpl_example_content`? :grinning:

Reply:
Heh. I initially implemented default opening hours in `hook_modules_installed()` because the code depends on the availability of default content which in itself is created by `hook_modules_installed()` implemented by `default_content`.

Perhaps the situation is the same for the front page since the alias `/delingstjenesten` will only exist if the BNF example content has been enabled.